applications/MeshingApplication/tests/cpp_tests/test_metric_process.cpp

```diff
@@ -303,11 +304,12 @@ namespace Kratos
                 auto it_node = this_model_part.Nodes().begin() + i_node;
                 it_node->FastGetSolutionStepValue(DISTANCE) = (it_node->X() == 1.0) ? 0.0 : 1.0;
                 it_node->SetValue(NODAL_H, 1.0);
+                it_node->SetValue(NODAL_AREA, 0.0);
                 it_node->SetValue(METRIC_TENSOR_3D, ZeroVector(6));
             }
 
             // Compute gradient
-            typedef ComputeNodalGradientProcess<3, Variable<double>, Historical> GradientType;
+            typedef ComputeNodalGradientProcess<ComputeNodalGradientProcessSettings::SaveAsHistoricalVariable> GradientType;
```

Comment:
why not one test with historical and the other with non-historical?

Reply:
Right now the level set process only considers historical values